.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "Sources/Server/Transcripts"]
-	path = Sources/Server/Transcripts
+	path = Sources/PointFree/Transcripts
```

Comment:
This stuff can live in the `PointFree` module now, which means our bootstrap script can in full, as well.